contracts/standard/arbitration/CentralizedCourt.sol

```diff
@@ -0,0 +1,231 @@
+     *  @param _extraData Can be used to give additional info on the dispute to be created.
+     *  @return disputeID ID of the dispute created.
+     */
+    function createDispute(uint _choices, bytes _extraData) public payable returns(uint disputeID)  {
+        require(msg.value >= arbitrationPrice);
+
+        disputeID = disputes.length++;
+        Dispute storage dispute = disputes[disputeID];
+        dispute.deadline = now + 3 days;
+        dispute.choices = _choices;
+        dispute.arbitrated = Arbitrable(msg.sender);
+        dispute.fee = msg.value;
+
+
+        emit DisputeCreation(disputeID, Arbitrable( msg.sender));
+        return disputeID;
+    }
+
+    /** @dev Give a ruling. UNTRUSTED.
+     *  @param _disputeID ID of the dispute to rule.
+     *  @param _ruling Ruling given by the arbitrator. Note that 0 means "Not able/wanting to make a decision".
+     */
+    function voteRuling(uint _disputeID, uint _ruling) public onlyCourtMember whoHasNotVoted(_disputeID) beforeDeadline(_disputeID) {
+        Dispute storage dispute = disputes[_disputeID];
+        require(_ruling<=dispute.choices && _ruling != 0);
+        require(dispute.status == DisputeStatus.Waiting);
+
+        dispute.hasVoted[msg.sender] = true;
+        dispute.voteCount[_ruling] += 1;
+
+        dispute.votes.push(
+            Vote({
+                account: msg.sender,
+                ruling: _ruling
+            })
+        );
+
+        VoteCounter storage voteCounter = dispute.voteCounter;
+        if(dispute.voteCount[_ruling] > voteCounter.winningCount){
+            voteCounter.winningCount = dispute.voteCount[_ruling];
+            voteCounter.winningChoice = _ruling;
+        }  else if (voteCounter.winningCount == dispute.voteCount[_ruling]) {
+            voteCounter.winningChoice = 0; // It's currently a tie.
+        }
+
+        if(dispute.voteCount[_ruling] >= (members.length/2)+1) { // We have got the will of the majority
+            for(uint8 i = 0; i < members.length; i++){
+                dispute.canCollectFee[members[i]] = true;
+            }
+            executeRuling(_disputeID);
+        }
+    }
+
+    /** @dev Do all the internal work and executes the rule function
+    *   @param _disputeID ID of the dispute to rule
+    */
+    function executeRuling(uint _disputeID) internal {
+        Dispute storage dispute = disputes[_disputeID];
+        VoteCounter storage voteCounter = dispute.voteCounter;
+        Arbitrable arbitrated = dispute.arbitrated;
+        dispute.arbitrated = Arbitrable(0x0); // Clean up to get gas back and prevent calling it again.
+        dispute.status = DisputeStatus.Solved;
+
+        arbitrated.rule(_disputeID, voteCounter.winningChoice);
+    }
+
+    /** @dev Calculates and tranfers the fee to the court member
+    *   @param _disputeID ID of the dispute to rule 
+    */
+    function withdraw(uint _disputeID) public onlyCourtMember {
```

Comment:
Using this, they need to withdraw on each dispute, it would be better keep the amount of wei each arbitrator has and having a withdraw function without any parameter withdrawing everything.